keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -208,21 +208,21 @@
           "Keys"
         ],
         "operationId": "GetKey",
-        "description": "Retrieves the public portion of a key plus its attributes.",
+        "description": "Gets the public part of a stored key. The get key operation is applicable to all key types. If the requested key is symmetric, then no key material is released in the response. Authorization: Requires the keys/get permission.",
         "parameters": [
           {
             "name": "key-name",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the key."
+            "description": "The name of the key to get."
           },
           {
             "name": "key-version",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The version of the key."
+            "description": "Adding the key-version parameter retrieves a specific version of a key."
```

Comment:
Instead of key-version use version of the key because the name changes to keyVersion in the generated code.